src/test/regress/expected/outer_join_pushdown.out

```diff
@@ -0,0 +1,83 @@
+-- ===================================================================
+-- test complex outer join trees 
```

Comment:
we also have multi_outer_join test. would it make sense to add these tests there ?
outer join pushdown tests already spread out to multiple files. since we only introduce 4 queries here. we might consider appending these tests to another file such as multi_outer_join.

Reply:
makes sense, that file was not there when I opened this PR.